src/config/index.ts

```diff
@@ -47,4 +47,8 @@ export const config = {
     url: 'wss://relay.walletconnect.com',
     projectId: '299fda67fbf3b60a31ba8695524534cd',
   },
+  aclhemyPay: {
```

Comment:
Nit: typo

Reply:
I might need a pair of glasses :eyes:  :eyeglasses: